spec/system/label.spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+RSpec.describe 'ラベル機能', type: :system do
+  describe 'ラベル作成機能' do
+    before do
+      user_a = FactoryBot.create(:user, user_name: 'ユーザーA', email: 'a@example.com')
+      FactoryBot.create(:task, name: '最初のタスク', detail: 'テスト１', deadline: '2020/08/10', status: '未着手', priority: '低', user: user_a)
+      FactoryBot.create(:second_task, name: '二番目のタスク', detail: 'テスト２', deadline: '2020/08/20', status: '着手中', priority: '中', user: user_a)
+      FactoryBot.create(:third_task, name: '三番目のタスク', detail: 'テスト３', deadline: '2020/08/30', status: '完了', priority: '高', user: user_a)
```

Comment:
`FactoryBot`という記載は設定ファイルに追記を行うことで省略できるようになります。
コード量が大分減るので、ぜひ調べてみてください。
```suggestion
      user_a = create(:user, user_name: 'ユーザーA', email: 'a@example.com')
      create(:task, name: '最初のタスク', detail: 'テスト１', deadline: '2020/08/10', status: '未着手', priority: '低', user: user_a)
      create(:second_task, name: '二番目のタスク', detail: 'テスト２', deadline: '2020/08/20', status: '着手中', priority: '中', user: user_a)
      create(:third_task, name: '三番目のタスク', detail: 'テスト３', deadline: '2020/08/30', status: '完了', priority: '高', user: user_a)
```

Reply:
調べますので保留にさせていただきたく思います。